rdkit/Chem/MolDb/FingerprintUtils.py

```diff
@@ -86,18 +85,17 @@ def BuildTorsionsFP(mol):
 
 
 def BuildRDKitFP(mol):
-    fp = Chem.RDKFingerprint(mol, nBitsPerHash=1)
-    return fp
+    return Chem.RDKFingerprint(mol, nBitsPerHash=1)
 
 
 def BuildPharm2DFP(mol):
     global sigFactory
     from rdkit.Chem.Pharm2D import Generate
     try:
         fp = Generate.Gen2DFingerprint(mol, sigFactory)
-    except IndexError:
+    except IndexError as e:
```

Comment:
I don't understand this change. Why introduce the `as` if you aren't going to use the exception?
Note that the original code matches the pattern shown in the python docs (second code example here: https://docs.python.org/3/tutorial/errors.html#raising-exceptions)

Reply:
Thank you for your review. I will revert back into original